src/ui/reducers/timeline.ts

```diff
@@ -92,46 +92,12 @@ export const {
   setPlaybackPrecachedTime,
   setPlaybackStalled,
   setFocusRegion,
+  setDisplayedFocusRegion,
   setTimelineState,
   pointsReceived,
   paintsReceived,
 } = timelineSlice.actions;
 
-export const pointsReceivedThunk = (points: TimeStampedPoint[]): UIThunkAction => {
```

Comment:
Hmmm... if we don't do this at all, then we are going to have bugs around analysis windows when points come in at times outside the displayed time window but inside the focus window points. Am I missing something there?